src/robusta/core/sinks/robusta/robusta_sink.py

```diff
@@ -191,7 +192,7 @@ def __to_node_info(cls, node: Node) -> Dict:
 
     @classmethod
     def __from_api_server_node(
-        cls, api_server_node: Node, pod_requests: List[PodRequests]
+        cls, api_server_node: Node, pod_requests: List[PodResources]
```

Comment:
confusing name for pod_requests list and pod_requests function we import

Reply:
if we didnt use both on this sinc i would say its no issue, would just rename to pod_requests_list

Reply:
fixed